.github/workflows/2.6.yml

```diff
@@ -64,4 +60,5 @@ jobs:
         brew tap-new $USER/local-autoconf
         brew extract --version=2.69 autoconf $USER/local-autoconf
         brew install autoconf@2.69
+        echo "/usr/local/opt/autoconf@2.69/bin" >> $GITHUB_PATH
```

Comment:
Since the autoconf fix has been merged, do we still need to do this? Can we just use autoconf 2.71 instead?

Reply:
What do you think about just pointing it at that commit instead of the commit pointed at by the 2.7.3 tag for now?

Reply:
That's a good idea. I can probably try that out in the next few days.